src/test/kotlin/racingcar_winner/CarFactoryTest.kt

```diff
@@ -0,0 +1,47 @@
+package racingcar_winner
+
+import org.assertj.core.api.Assertions.assertThat
+import org.assertj.core.api.Assertions.assertThatThrownBy
+import org.junit.jupiter.api.Test
+import org.junit.jupiter.params.ParameterizedTest
+import org.junit.jupiter.params.provider.Arguments
+import org.junit.jupiter.params.provider.Arguments.arguments
+import org.junit.jupiter.params.provider.MethodSource
+import racingcar_winner.model.Car
+import racingcar_winner.model.CarFactory
+import racingcar_winner.model.Cars
+import java.lang.IllegalArgumentException
+import java.util.stream.Stream
+
+class CarFactoryTest {
+
+    private val carFactory = CarFactory()
+
+    @ParameterizedTest
+    @MethodSource("orderCarParameters")
+    fun `이름들을 전달 받았을 때 차들을 생성`(carNames: List<String>, expected: Cars) {
+        assertThat(carFactory.orderMakingCars(carNames).toString()).isEqualTo(expected.toString())
+    }
+
+    @Test
+    fun `이름중에 5글자 이상인 글자가 존재`() {
```

Comment:
테스트 이름에 명확하게 **~할 때, ~해야 한다**를 밝혀주면 조금 더 가독성있는 코드가 될 것 같아요.

```suggestion
    fun `이름중에 5글자 이상인 글자가 존재하면 에러가 발생해야 한다`() {
```